asistente_ladm_col/gui/qgis_model_baker/dlg_import_data.py

```diff
@@ -195,7 +202,7 @@ def update_schema_names_model(self):
             default_item = self.schema_names_list_widget.item(0)
             default_item.setCheckState(Qt.Checked)
         else:
-            message_error = "There are no schemata to import into the database. Select another database."
+            message_error = "There are no schema to import into the database. Select another database."
```

Comment:
The plural form is 'schemata' :P

Reply:
I didn't know that. You learn something every day.